tendrl/commons/utils/etcd_utils.py

```diff
@@ -0,0 +1,68 @@
+import etcd
+import __builtin__
+
+'''
+   Read from etcd
+   input params:
+       key  : type  - >  string
+              value - >  attr to be fetched
+
+   return param:
+       dict - > if read is successful
+       None - > if key is not found
+'''
+ 
+def read(key):
+    try:
+        return NS._int.client.read(key)
+    except (etcd.EtcdConnectionFailed, etcd.EtcdException) as ex:
```

Comment:
So, as we discussed this won't be required as the handler can still catch such exceptions.... even with the current approach...